python/tvm/topi/adreno/conv2d_transpose_nchw.py

```diff
@@ -0,0 +1,383 @@
+            tvm.tir.all(
+                x >= pad_left,
+                x < pad_left + dilated_width,
+                tvm.tir.indexmod(x - pad_left, stride_w).equal(0),
+                y >= pad_top,
+                y < pad_top + dilated_height,
+                tvm.tir.indexmod(y - pad_top, stride_h).equal(0),
+            ),
+            Input[
+                n,
+                c,
+                tvm.tir.indexdiv(y - pad_top, stride_h),
+                tvm.tir.indexdiv(x - pad_left, stride_w),
+                cb,
+            ],
+            tvm.tir.const(0.0, Input.dtype),
+        ),
+        name="pad_temp",
+    )
+
+    # compute transposed conv
+    dcc = te.reduce_axis((0, in_channel_chunks), name="dcc")
+    dcb = te.reduce_axis((0, in_channel_block), name="dcb")
+    dh = te.reduce_axis((0, kernel_h), name="dh")
+    dw = te.reduce_axis((0, kernel_w), name="dw")
+    conv = te.compute(
+        (batch, out_channel_chunks, out_height, out_width, out_channel_block),
+        lambda b, c, h, w, cb: te.sum(
+            temp[
+                b, c // out_channel_chunks * (in_channel_chunks) + dcc, h + dh, w + dw, dcb
+            ].astype(out_dtype)
+            * Filter[
+                dcc * in_channel_block + dcb,
+                c % out_channel_chunks,
+                kernel_h - 1 - dh,
+                kernel_w - 1 - dw,
+                cb,
+            ].astype(out_dtype),
+            axis=[dcc, dcb, dh, dw],
+        ),
+        tag="conv2d_transpose_nchwc",
+    )
+
+    if convert_from4d and not autotvm.GLOBAL_SCOPE.in_tuning:
+        dummy_cast = te.compute(
+            (batch, out_channel_chunks, out_height_orig, out_width_orig, out_channel_block),
+            lambda n, fc, y, x, fb: conv[n, fc, y, x, fb].astype(out_dtype),
+            tag="dummy_cast",
+        )
+        return te.compute(
+            (batch, out_channels, out_height_orig, out_width_orig),
+            lambda n, c, y, x: dummy_cast[n, c // out_channel_block, y, x, c % out_channel_block],
+            tag="adreno_conv2d_transpose_latest_op",
+        )
+    else:
+        return te.compute(
+            (batch, out_channel_chunks, out_height_orig, out_width_orig, out_channel_block),
+            lambda n, ffc, y, x, ffb: conv[n, ffc, y, x, ffb].astype(out_dtype),
+            tag="adreno_conv2d_transpose_latest_op",
+        )
+
+
+@autotvm.register_topi_schedule("conv2d_transpose_nchwc.image2d")
+def schedule_conv2d_transpose_nchwc(cfg, outs):
+    """Create the schedule for conv2d_nchw"""
+    outs = [outs] if isinstance(outs, te.tensor.Tensor) else outs
+    s = te.create_schedule([x.op for x in outs])
+
+    def _callback(op):
+        if op.tag == "adreno_conv2d_transpose_latest_op":
+            schedule_conv2d_transpose_NCHWc(cfg, s, op.output(0))
+
+    traverse_inline(s, outs[0].op, _callback)
+    return s
+
+
+def schedule_conv2d_transpose_NCHWc(cfg, s, output):
+    """
+    schedule optimized for batch size = 1
+
+    Algo:
```

Comment:
The same question about `Algo` here.

Reply:
I have added the description of algorithm.
Thanks for review.